### PR TITLE
Support use of "docker-compose run".

### PR DESCRIPTION
**Pull request by manarth**
### Overview
This pull request:

| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| Has tests?    | no
| BC breaks?    | no     
| Deprecations? | no 

### Summary
Supports the use of `docker-compose run`, to invoke commands on non-running docker containers.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,6 +80,9 @@ docker-compose --project dockerComposeProjectName --file docker-compose.yml --pr
 
 `docker.service` is the exact name of the service as it appears in docker-compos.yml
 
+The default behaviour is to use `docker-compose exec` to invoke a command on a running container.
+You can change this to use `docker-compose run` for containers which by default are not running, with the property `composer.command`.
```
Comment (greg-1-anderson) [inline README.md:84]: "composer.command" should be "compose.command"

Comment (manarth) [reply README.md]: Thanks for the spot – fixed (and also the typo in the line above, `docker-compos.yml`).

**Comment by manarth** [comment]
> Would you mind fixing the backward compatibility check while you're in here?

Sure – done. I've updated both the `composer.json` in the repo, and the steps in the CI workflow to fix and run the backwards-compatibility test, and all tests are passing.

**Comment by greg-1-anderson** [comment]
Thanks for your contribution and help.

**Pull request merged**